### DISCUSSION TITLE
switch to the microsoft store app (while keeping my backgrounds)

### DISCUSSION THREAD

**Discussion by Nectarinnie** [rocksdanister/lively#535] [Q&A]
How can I switch from the normal app to the microsoft store app? While keeping all my backgrounds, of course! I know that when I uninstall it, I'm losing everything. I've uninstalled it once because Windows only let me **replace** it with the newer version like this. I want to switch because of that, so I don't have to uninstall anymore, but I'll probably have to do so one last time because I can't download from the microsoft store otherwise.

**Comment by rocksdanister** [comment]
I don't understand, why do you want to switch to store version?

**Reply by Nectarinnie** [reply]
Because when there's an update of the normal app, I have to uninstall the old version everytime. If I had the store version, it would update automatically. And as I said, I'm losing all my backgrounds when I uninstall the app. :(

**Reply by rocksdanister** [reply]
What? That is not needed.. 
Lively has built in updater, just exit the Lively from traymenu and start again, it will prompt you or restart system and you will get a prompt or if you are downloading the installer yourself then exit Lively from traymenu before running it.

**Reply by Nectarinnie** [reply]
huh, somehow it worked now! It wasn't earlier, but now it's all good. Thank you :) 💯
